### ISSUE TITLE
1.7.32 breacking change - countryCallingCode

### ISSUE THREAD

**Issue by cuzik** [catamphetamine/libphonenumber-js#388]
## Steps to reproduce

```js
formatter = libphonenumber.AsYouType("BR")
formatter.countryCallingCode
```

## Observed result

this returns `undefined`.

## Expected result

before the path version this code returns `"55"`

**Comment by catamphetamine** [comment]
Hi.
`countryCallingCode` is not part of the public API:
https://github.com/catamphetamine/libphonenumber-js#class-asyoutypeoptions-or-defaultcountry
Perhaps it has been changed at some point, but not recently.

**Comment by catamphetamine** [comment]
A note: `.countryCallingCode` is currently only set when an international number is being input (with a `+`).
When the user doesn't input `+`, `.countryCallingCode` isn't set.
If you think there's a valid use case when a developer might want to have access to `.countryCallingCode` (that isn't part of the API) then you can post a comment.

**Comment by catamphetamine** [comment]
So I was thinking on adding a couple of utility methods to `AsYouType` now that this topic has been raised.
Currently, it only has `.getNumber()` and `.getTemplate()`, and `.getNumber()` only returns non-`undefined` when there's at least a single national number digit been input.
Some developers might want something like:

```js
	/**
	 * Returns `true` if the phone number is being input in international format.
	 * In other words, returns `true` if and only if the parsed phone number starts with a `"+"`.
	 * @return {boolean}
	 */
	isInternational()

	/**
	 * Returns the parsed phone number.
	 * Returns `undefined` if no phone number digits have been input.
	 * (`"+"` is not considered a "digit").
	 * Returned value examples: `undefined`, `"+1"`, `"+1213"`, `"213"`.
	 * @return {string} [value]
	 */
	getValue()

	/**
	 * Returns the "country calling code" part of the phone number.
	 * Returns `undefined` if the number is not being input in international format.
	 * Returns "country calling code" for "non-geographic" phone numbering plans too.
	 * @return {string} [countryCallingCode]
	 */
	getCountryCallingCode()

	/**
	 * Returns a two-letter country code of the phone number.
	 * Returns `undefined` for "non-geographic" phone numbering plans.
	 * Returns `undefined` if no phone number has been input yet.
	 * @return {string} [country]
	 */
	getCountry()

	/**
	 * Returns `true` if the phone number is "possible".
	 * Is just a shortcut for `PhoneNumber.isPossible()`.
	 * @return {boolean}
	 */
	isPossible()

	/**
	 * Returns `true` if the phone number is "valid".
	 * Is just a shortcut for `PhoneNumber.isValid()`.
	 * @return {boolean}
	 */
	isValid()
```

@cuzik 
What are your thoughts?
And why did you read `countryCallingCode` property?

**Comment by catamphetamine** [comment]
Published `libphonenumber-js@1.7.50`.
